dist.ini

```diff
@@ -7,6 +7,8 @@ copyright_holder = MaxMind, Inc.
 dist = WebService-PivotalTracker
 stopwords = distro
 Test::CleanNamespaces.skip = ^WebService::PivotalTracker::(?:PropertyAttributes|Types)$
+; Our fixtures have some weird file names
```

Comment:
Is this something that should be fixed?

Reply:
Maybe, but that would require restructuring how fixtures are handled. I don't think this particular filename will cause issues on any platforms we care about. It's only an issue on things like VMS or platforms that don't support long file names.

Reply:
I'll add an `appveyor.yml` just to get this tested on another platform.

Reply:
But for some reason appveyor is not picking up this branch :(

I wonder if it needs to have run master at least once ...

Reply:
Nope, can't get it to run. Presumably it'll run once this is merged.